inst/tinytest/test-datetime.R

```diff
@@ -1,5 +1,6 @@
 source("helpers.R")
 using("tinysnapshot")
+if (ON_CRAN) exit_file("CRAN")
 
```

Comment:
I might just be missing something obvious, but is there a reason the ON_LINUX  flags aren't included in some of these files?

Reply:
I guess they could be, but you don't have any other OS in the github actions matrix, and you probably only run the tests on a linux machine, so it feels superfluous.

Reply:
I know it cuts against the tinytest ethos, but I wonder whether she should just add these flags to the `helpers.R` file, so that they are automatically enabled for all test scripts (instead of adding individually to each one). Thoughts?

Reply:
Sure. Or include a similar logic in `/tests/tinytest.R` do skip the whole run.